firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -299,35 +298,34 @@ public TokenResult generateAuthToken(
 
         return TokenResult.builder().setResponseCode(TokenResult.ResponseCode.BAD_CONFIG).build();
       } finally {
-        httpsURLConnection.disconnect();
+        httpURLConnection.disconnect();
       }
     }
     throw new IOException();
   }
 
-  private HttpsURLConnection openHttpsURLConnection(URL url) throws IOException {
-    HttpsURLConnection httpsURLConnection = (HttpsURLConnection) url.openConnection();
-    httpsURLConnection.setConnectTimeout(NETWORK_TIMEOUT_MILLIS);
-    httpsURLConnection.setReadTimeout(NETWORK_TIMEOUT_MILLIS);
-    httpsURLConnection.addRequestProperty(CONTENT_TYPE_HEADER_KEY, JSON_CONTENT_TYPE);
-    httpsURLConnection.addRequestProperty(ACCEPT_HEADER_KEY, JSON_CONTENT_TYPE);
-    httpsURLConnection.addRequestProperty(CONTENT_ENCODING_HEADER_KEY, GZIP_CONTENT_ENCODING);
-    httpsURLConnection.addRequestProperty(X_ANDROID_PACKAGE_HEADER_KEY, context.getPackageName());
+  private HttpURLConnection openHttpURLConnection(URL url) throws IOException {
+    HttpURLConnection httpURLConnection = (HttpURLConnection) url.openConnection();
```

Comment:
@gsakakihara  Can this be HttpsURLConnection or will that cause the TAP failures too? I have concerns on using Http over Https.